pages/code_schools.js

```diff
@@ -46,7 +48,23 @@ export default class CodeSchools extends React.Component {
         </Section>
         <Section theme="gray" title="Schools" hasHeadingLines={false}>
           {state.schools.map(school => (
-            <div key={`${Math.random()}`}>{school.name}</div>
+            <div key={`${Math.random()}`}>
```

Comment:
I think you'll learn something new when reading [this](https://stackoverflow.com/questions/29808636/when-giving-unique-keys-to-components-is-it-okay-to-use-math-random-for-gener).

I recommend using `school.name` as the key.